common/src/androidTest/java/com/microsoft/identity/common/adal/internal/cache/StorageHelperTests.java

```diff
@@ -186,6 +187,31 @@ public void testEncryptSameText() throws GeneralSecurityException, IOException {
         Log.d(TAG, "Finished testEncryptSameText");
     }
 
+    @Test
+    public void testKeyThumbprint() throws GeneralSecurityException, IOException {
+        Context context = getInstrumentation().getTargetContext();
+        final StorageHelper storageHelper = new StorageHelper(context);
+        for (int i = 0; i < 1000; i++) {
```

Comment:
```suggestion
```

Reply:
okay this then resolves my comment thanks :)